### DISCUSSION TITLE
chapter_dynamic_programming/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#572] [Announcements]
# chapter_dynamic_programming/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_dynamic_programming/

**Comment by shanghai-Jerry** [comment]
每次有新的内容更新的时候， 要怎么通知到读者会比较好？

**Reply by krahets** [reply]
我也在思考这个问题，感觉需要建立一个社群

**Reply by shanghai-Jerry** [reply]
感觉简单一点的做法，应该可以周期性的发布release， 然后里面列举出本次发布中新增的内容。 
目前的话，一点点往里新增内容是可以不发布release，等到有一定量的新内容时，在发布release应该就可以了

**Reply by krahets** [reply]
我觉得 release page 的关注度不如网页版主页的关注度，目前新章节会在网页版标注（New）。

最大的问题是如何更有效地“通知”到读者。

**Reply by shanghai-Jerry** [reply]
嗯， 这个New是相对上一个release之后的么？

**Reply by krahets** [reply]
一大部分是的

**Reply by MeetYou2022** [reply]
还是建个社群，Follow一下。。

**Reply by krahets** [reply]
@MeetYou2022 可以加我微信 krahets-jyd

**Comment by suson-struggle** [comment]
卡神，代码随想录和这个hello算法应该看哪个呀，还是说两个都看，有顺序推荐吗？

**Reply by krahets** [reply]
Hi 可以把两个都先读一读，感觉哪个读的顺就先读哪个～

**Comment by rookiexxj** [comment]
大神怎么更新强化学习了

**Reply by krahets** [reply]
Hi 哪里更新了？🤫

**Comment by vanyongqi** [comment]
真的很不错

**Comment by curiosity2** [comment]
能不能每个知识点给一些对应的leetcode题目的链接

**Reply by krahets** [reply]
推荐以下 LeetCode 题单（前两项 [@Krahets](https://leetcode.cn/u/jyd/) 题解全覆盖）：

1. [Krahets 笔面试精选 88 题](https://leetcode.cn/studyplan/selected-coding-interview/)：从“剑指 Offer”和“热题 100”精选出的 88 道高频算法笔试题。
2. [图解算法数据结构](https://leetcode-cn.com/leetbook/detail/illustration-of-algorithm/)（[Interview-75](https://leetcode.cn/studyplan/coding-interviews/)）：题目更贴近实际应用，相对“剑指 Offer”难度有所增加。
3. [LeetCode 热题 100](https://leetcode.cn/studyplan/top-100-liked/)：力扣用户最喜爱的 100 道题。
4. [面试经典 150 题](https://leetcode.cn/studyplan/top-interview-150/)：150 道经典面试力扣题。
5. [LeetCode-75](https://leetcode.cn/studyplan/leetcode-75/)：精选面试必备的 75 道核心题目。

**Reply by lizujin-magictavern-com** [reply]
mark一下

**Reply by emlynma** [reply]
mark

**Reply by zengfantaoGit** [reply]
mark

**Reply by correcculun** [reply]
mark一下

**Comment by UselessWater** [comment]
有个不成熟的建议，如果有机会，希望能与一些做视频教程做的好的合作，根据这本书来做讲解。原因：以本人读这本书的体验下来，感觉很适合新手，就是对于很多人来说，文字来理解，时间长了，会让人有一种很难受的感觉，兴趣也会大大降低。这本书我看了三个月左右，如果是视频内容，我觉得只需要两个月不到就可以学完。新手对于繁琐的文字，真的很难坚持下去，如果有机会，强力建议！

**Reply by krahets** [reply]
Hi，可以关注一下我的 Bilibili 账号：[K 源算法](https://space.bilibili.com/196980)

**Comment by MambaFly** [comment]
终于！要到动态规划啦，激动！
